src/TrelloUtilities.py

```diff
@@ -1,17 +1,34 @@
 import json
+
+'''
 from datetime import datetime
 import pytz
 import TrelloMembers
 import TrelloBoards
+'''
+
+class Credentials():
+
+    key = ''
+    token = ''
+
+    def __init__(self):
+
+        # open secrets.json to get key and token
+        if self.key == '' or self.token == '':
+            with open("../config/secrets.json", "r") as secrets_file:
 
+                secrets = json.load(secrets_file)
+                self.key = secrets['key']
+                self.token = secrets['token']
+
+'''
 def get_secrets():
```

Comment:
Remove this function.  Duplicates functionality introduced by Credentials class.